hiclass/Explainer.py

```diff
@@ -189,6 +189,46 @@ def _get_traversed_nodes_lcppn(self, samples):
                         ).flatten()
         return traversals
 
+    def _get_traversed_nodes_lcpn(self, samples):
+        """
+        Return a list of all traversed nodes as per the provided LocalClassifierPerNode model.
+
+        Parameters
+        ----------
+        samples : array-like
+            Sample data for which to generate traversed nodes.
+
+        Returns
+        -------
+        traversals : list
+            A list of all traversed nodes as per LocalClassifierPerNode (LCPN) strategy.
+        """
+        traversals = np.empty(
+            (samples.shape[0], self.hierarchical_model.max_levels_),
+            dtype=self.hierarchical_model.dtype_,
+        )
+
+        predictions = self.hierarchical_model.predict(samples)
+
+        traversals[:, 0] = predictions[:, 0]
+        separator = np.full(
+            (samples.shape[0], 3),
+            self.hierarchical_model.separator_,
+            dtype=self.hierarchical_model.dtype_,
+        )
+
+        for level in range(1, traversals.shape[1]):
+            traversals[:, level] = np.core.defchararray.add(
```

Comment:
Hi @ashishpatel16 
I was checking numpy's documentation and it seems defchararray is not the recommended way to do concatenations. Can you please update to numpy.char? Please see https://numpy.org/doc/stable/reference/generated/numpy.char.chararray.html and https://numpy.org/doc/stable/reference/routines.char.html#module-numpy.char

Reply:
Hi @mirand863, Yes, you are correct! I have now used `numpy.char` for string concatenations as per the numpy documentation. 

Resolved by https://github.com/scikit-learn-contrib/hiclass/pull/108/commits/5aa9998664ec22843cae47c12c423087730930ab.